app/person/admin.py

```diff
@@ -1,5 +1,18 @@
-from django.contrib import admin
-
+from django.contrib import admin, messages
+from rollup.utils import load_rollup
+from django_object_actions import DjangoObjectActions
 from .models import Person
-# Register your models here.
-admin.site.register(Person)
+
+# Adds a button to the Person Admin page to allow users to manually update the
+# directory screens with the most up-to-date data
+class PersonAdmin(DjangoObjectActions, admin.ModelAdmin):
+
+  def UpdateScreens(self, request, queryset):
+    load_rollup()
```

Comment:
This should be wrapped in a `try ... except` block to more gracefully handle errors when updating the index (presumably with a different `message_user` call that displays an error message).

Reply:
Good point! I updated the function to include a `try...except` clause in fa713dc.